### DISCUSSION TITLE
DIA_NN identification strategy

### DISCUSSION THREAD

**Discussion by kk1058698** [vdemichev/DiaNN#908] [Q&A]
hi,
My reviewer ask a question about which criteria were useed to filter protein and peptide identification? and which protein interference strategy was used in the DIA_NN?
how can I accurateely respond to these questions?
we are eager for your assistance.

many thanks

**Comment by vdemichev** [comment]
Which filtering criteria - depends on how you filtered the DIA-NN main report. DIA-NN's documentation contains a detailed guide on this. For an introduction on different q-value concepts in proteomics I can recommend https://www.nature.com/articles/nmeth.4398. As for the general principles DIA-NN uses:
- peptide identification is desribed in https://www.nature.com/articles/s41592-019-0638-x
- protein inference is based on the maximum parsimony principle, implemented similarly (but not identically) to what's desribed in the IDPicker paper https://pubs.acs.org/doi/abs/10.1021/pr900360j.

**Reply by kk1058698** [reply]
thanks for your detail answers, and I would like to confirm with you if I can describe the identification process of DIA_NN in the method as follows: proteins were identified based on a 2-peptide minimum and an FDR 0.01 %. If this is not accurate about the peptide number, how can I optimize it?
many thanks

**Reply by vdemichev** [reply]
DIA-NN does not apply '2 peptide minimum'.

**Reply by kk1058698** [reply]
get it, thanks!
